be/app/Http/Controllers/TasksController.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
- remove this file. this is not related to the current task (apply to all related)